Jenkinsfile

```diff
@@ -1,49 +1,20 @@
 #!groovy
-@Library('waluigi@v5.0.0') _
-
-standardProperties()
-
-node("primary") {
-  echo "Clean workspace"
-  cleanWs()
-
-  stage("checkout") {
-    checkout localBranch(scm)
-  }
-
-  stage("dependencies") {
-    yarnInstall()
-  }
-
-  stage("stamp") {
-    sh "yarn beehive-flow stamp"
-  }
-
-  stage("build") {
-    sh "yarn build"
-  }
-
-  stage("lint") {
-    sh "yarn lint"
-  }
-
-  bedrockBrowsers()
-
-  stage("update storybook") {
-    def status = beehiveFlowStatus();
-    if (status.branchState == 'releaseReady' && status.isLatest) {
-      sshagent (credentials: ['jenkins2-github']) {
-        sh 'yarn deploy-storybook'
+@Library('waluigi@v6.0.0-rc') _
```

Comment:
This is just a temporary tag I've setup so that we won't have to change this as much (or at least that's the goal may not end up being effective though).

Reply:
I was going to ask about the version when I saw the Vue PR

Reply:
Yeah, the problem is it's not quite stable yet so I'm hesitant to merge to `main` and tag as `v6.0.0`. I'm planning to do that once I get through a few more conversions and see what's missing.

Reply:
This has been updated and it's now using the tagged 6.0.0 version.